src/store/reducers/dform/initial/styles.json

```diff
@@ -3140,6 +3140,70 @@
             }
           ]
         },
+        {
+           "class": "block",
+           "label": "SEMEN ANALYSIS",
+           "children":[
+             {
+               "field": "Pathology.semen.analysis.collectionTime",
```

Comment:
I would like to state the same thing about here to https://github.com/asha-nepal/AshaFusionCross/pull/671#pullrequestreview-138219473
Please check that.